generator/src/main/java/com/algorand/sdkutils/utils/Tools.java

```diff
@@ -109,4 +116,53 @@ public static String getCamelCase(String name, boolean firstCap) {
         }
         return new String(newName, 0, n);
     }
+
+    // Imports are collected and organized before printed as import statements.
+    // addImports adds a needed import class. 
+    public static void addImport(Map<String, Set<String>> imports, String imp) {
```

Comment:
Not sure it will "break". 
It depends how we want them organized. They will still be grouped together nicely, but the order might not be what you want. 
Anyway, complexity is contained within the Tools class.